### PR TITLE
DCJ-495: New Cert documentation and configuration render script

### PR DESCRIPTION
**Pull request by rushtong**
### Addresses
https://broadworkbench.atlassian.net/browse/DCJ-495

### Summary
Updates documentation for how to generate certificates for local development
Also adds a script to do the rendering for local dev needs.

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: scripts/render-configs.sh [hunk 1]
```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+# Populates local cert files necessary for local development
+# You MUST be on the Broad VPN
+# You MUST have gcloud and kubectl installed to run this script.
```
Comment (fboulnois) [inline scripts/render-configs.sh]: And `jq`

File: scripts/render-configs.sh [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+}
+
+auth_gcloud() {
+  echo "Getting cluster credentials"
+  gcloud container clusters get-credentials --zone us-central1-a --project "$PROJECT" terra-dev
+}
+
+write_certs() {
+  echo "Writing cert files"
+  kubectl -n local-dev get secrets local-dev-cert -o 'go-template={{ index .data "tls.crt" | base64decode }}' > ../server.crt
```
Comment (fboulnois) [inline scripts/render-configs.sh:67]: This is neat.

**Review by fboulnois — APPROVED** [review]
Looks good! 👍 Huge fan of simplifying local deployments.

**Review by okotsopoulos — APPROVED** [review]
Nice!

I'm remembering that you said these certificates will need to be regenerated more frequently moving forward (quarterly).  Should that be documented somewhere?

**Comment by rushtong** [comment]
> I'm remembering that you said these certificates will need to be regenerated more frequently moving forward (quarterly). Should that be documented somewhere?

Great point, yes, it's now on a 3-month rotation, I'll add that.

File: scripts/render-configs.sh (hunk 2 above)
Comment (rushtong) [reply scripts/render-configs.sh:67]: This is cribbed from the [leonardo setup script](https://github.com/DataBiosphere/leonardo/blob/3aabca9750e77df7c0639ae8eec0f5dbe2781345/local/depends.sh#L231)

**Pull request merged**